documentation/sections/changelog.md

```diff
@@ -1,6 +1,22 @@
 ## Change Log
 
 ```
+releaseHeader('2017-02-10', '2.0.0-alpha1', '1.12.3')
+```
+*   Initial alpha release of CoffeeScript 2. The CoffeeScript compiler now outputs ES2015+ syntax whenever possible.
```

Comment:
Perhaps we should say something like "The CoffeeScript compiler now _intends_ outputs ES2015+ syntax whenever possible." Currently, it sounds like CS2 is already done.